### PR TITLE
Upgrade alpine 3.16.4 -> 3.16.5

### PR DESCRIPTION
**Pull request by trevorwhitney**
**What this PR does / why we need it**:

Upgrade alpine version to address [openssl CVEs](https://www.alpinelinux.org/posts/Alpine-3.17.3-released.html)

**Checklist**
- [ ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

**Review by DylanGuedes — APPROVED** [review]

**Review by vlad-diachenko — APPROVED** [review]
LGTM. just a comment that we could resolve in a separate PR

**Review by dannykopping — APPROVED** [review]
Thanks!!

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +38,10 @@
 * [9185](https://github.com/grafana/loki/pull/9185) **dannykopping**: Prevent redis client from incorrectly choosing cluster mode with local address.
 * [9252](https://github.com/grafana/loki/pull/9252) **jeschkies**: Use un-escaped regex literal for string matching.
 
+##### Build
+
+* [9264](https://github.com/grafana/loki/pull/9264) **trevorwhitney**: Update build and other docker image to alpine 3.16.5.
```
[the remark this answers is not in the document]
Comment (DylanGuedes) [reply CHANGELOG.md:43]: good catch, looks like we forgot to update the changelog on main with the 2.8.1 changes

[the remark this answers is not in the document]
Comment (trevorwhitney) [reply CHANGELOG.md:43]: this is going into 2.8.1. The release branch has an updated changelog, but I'm waiting to bring it into main until after the release to help prevent the mess the changelog go into during the 2.8 release.

[the remark this answers is not in the document]
Comment (vlad-diachenko) [reply CHANGELOG.md:43]: I see ) thanks @trevorwhitney

**Pull request merged**